lib/IDE/CodeCompletionResultPrinter.cpp

```diff
@@ -118,9 +134,11 @@ class AnnotatingResultPrinter {
       printWithTag("typeid.user", C.getText());
       break;
     case ChunkKind::CallArgumentName:
+    case ChunkKind::ParameterDeclExternalName:
       printWithTag("callarg.label", C.getText());
```

Comment:
It's weird to reuse "callarg" when this is a declaration. Same for callarg.param and callarg.type. What's the reasoning here?

Reply:
Ah, this is another TODO part. This is temporary. I have to discuss with clients to decide what tags we should use for them.

Reply:
Updated:
* the base name is enclosed by `<name>...</name>` 
* single parameter declaration is enclosed by `<param> ... </param>`
* external label for a parameter is enclosed by `<param.label> ... </param.label>`
* internal name for a parameter is enclosed by `<param.param> ... </param.param>`
* types and keywords are annotated as the same as other completions